src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -119,30 +160,30 @@ public void findPersonInTransaction(Name name) throws PersonNotFoundException {
     }
 
     @Override
-    public Set<Transaction> findTransactionsWithPayer(Person person) throws TransactionNotFoundException {
+    public boolean findTransactionsWithPayer(Person person) throws PersonFoundException {
```

Comment:
I don't know why you replace this method with a boolean method. you could have created another one, called for exemple IsTransactionWithPerson(Person person) that will check at the same time payer and payees .